### PR TITLE
Adds media block

### LINKED ISSUE #1: Border Image Block
Need to make an image block with a customizable color and border, as per homepage design.

Designs:
Screen Shot 2020-04-21 at 11 24 04 PM

### PR DESCRIPTION
**Pull request by ChaosExAnima**
Fixes #1.

This PR adds a new custom Media Border block, used in the homepage.

To test:
1. Install dependencies with `yarn`.
2. Build block with `yarn build`.
3. Enjoy shiny new block!

Screen Shot 2020-04-22 at 10 09 14 PM

Todo:
* Add automatic linting of JS.
* Automated build and deploy of block assets.

### CODE REVIEW

File: blocks/media-border/style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	color: var(--color-dark-text);
+	border-color: var(--color-dark-border);
+}
+
+.wp-block-mutualaidnyc-media-border.has-dark-alt-color {
+	color: var(--color-dark-alt-text);
+	border-color: var(--color-dark-alt-border);
+}
+
+.wp-block-mutualaidnyc-media-border.has-accent-background-color {
```
Comment (jffng) [inline blocks/media-border/style.css:63]: This one conflicts with a 2020 class: 

Screen Shot 2020-04-23 at 6 46 44 PM

**Comment by ChaosExAnima** [comment]
@jffng - Fair, it's very similar to the Media & Text block. We could try and set up a style variation if you think that'd be a better approach? I imagine we can recycle the styles fairly easily.

**Comment by jffng** [comment]
Sure, I'll work up that PR and borrow from the styles here.

**Comment by ChaosExAnima** [comment]
Replaced by #15.

**Pull request closed without merging**